devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -99,6 +99,11 @@ def parse(String description) {
 		} else {
 			Map descMap = zigbee.parseDescriptionAsMap(description)
 			if (descMap?.clusterInt == 0x0001 && descMap.commandInt != 0x07 && descMap?.value) {
+				if (descMap?.attrInt == 0x003E) {
```

Comment:
you could instead log the `attrInt` and then you wouldn't need the if statement. It would also be helpful in case there is an unexpected attribute being reported.

ex: `log.info "BATT METRICS - ${descMap?.attrInt} attr; value: ${descMap?.value}, curr percent: ${device.currentState("battery")?.value}, device: ${device.getDataValue("manufacturer")} ${device.getDataValue("model")}"`

Reply:
@tpmanley That's actually not a bad idea. I added that, and the code has been updated.